docs/input/siemens-s7.md

```
@@ -0,0 +1,42 @@
+    tcpDevice: '192.168.0.1' # IP address of the S7 PLC
+    rack: 0                  # Rack number of the PLC. Defaults to 0
+    slot: 1                  # Slot number of the PLC. Defaults to 1
+    batchMaxSize: 480         # Maximum number of addresses per batch request. Defaults to 480
+    timeout: 10             # Timeout in seconds for connections and requests. Default to 10
+    disableCPUInfo: false # Set this to true to not fetch CPU information from the PLC. Should be used when you get the error 'Failed to get CPU information'
+    addresses:               # List of addresses to read from
+      - "DB1.DW20"     # Accesses a double word at location 20 in data block 1
+      - "DB1.S30.10"   # Accesses a 10-byte string at location 30 in data block 1
+```
+
+**Configuration Parameters**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)